docs/uml-class-diagrams/display02/production/DisplayCommInterfaceImplementation/DisplayCommInterfaceImplementation.puml

```diff
@@ -2,12 +2,14 @@
 
 package edu.oakland.production.display02 {
     class DisplayCommInterfaceImplementation {
+        DisplayComm2WayInterface receiver
         __
+        {abstract} + DisplayCommInterface(DisplayComm2WayInterface receiver)
         .. Use Case 1 ..
-
-        .. Use Case 2 ..
+        {abstract} + TrackData receiveRfidRequest(int rfidNum)
+        {abstract} + TrackData passToComm2WayInterface(int rfidNum)
```

Comment:
Remember: you're implementing this here, so it shouldn't be abstract. Also, are these private? If so, they should have the minus sign instead of the `+` sign.